lib/rubocop/lockfile.rb

```diff
@@ -5,14 +5,16 @@ module RuboCop
   # Does not actually resolve gems, just parses the lockfile.
   # @api private
   class Lockfile
-    # Gems that the bundle depends on
+    # Gems that the bundle directly depends on.
+    # @return [Array<Bundler::Dependency>, nil]
```

Comment:
These methods' semantics aren't obvious (in particular, whether they include indirect dependencies or not, and what type of results they vend), so I documented them all.

The whole class is marked `@api private`, which get inherited by all of these methods, so there's no risk of people confusing them for public API (even if they're documented here).